### DISCUSSION TITLE
Upgrade Problem

### DISCUSSION THREAD

**Discussion by wzzc-dev** [quarkusio/quarkus#29608] [Q&A]
2.3.1.Final -> 2.14.2.Final

Unit test happen error：
`Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE SEQUENCE hibernate_sequence [*]start 1 increment 1"; SQL statement:
CREATE SEQUENCE hibernate_sequence start 1 increment 1 [42000-214]`

**Comment by gsmet** [comment]
Are you somehow overriding either the H2 version or the Hibernate ORM version? Or do you specify a dialect yourself?

If you use all the standard versions, they should be compatible with each other and you shouldn't have this sort of errors.

**Reply by wzzc-dev** [reply]
I use the postgresql dialect, h2 2.x and then this problem comes up and I have to downgrade h2 and flyway

**Reply by gsmet** [reply]
You haven't answered my question. Are you overriding versions? Maybe share the relevant sections of your build file.

When using the Quarkus BOM, we provide versions that work well together. If you're overriding versions then you're on your own trying to find a working combination.

**Reply by wzzc-dev** [reply]
I am using version 2.14.2.Final, in which case flyway is h2 `jdbc:h2:mem:myDB; MODE=PostgreSQL`  does not work properly, the above error is displayed.

I saw this response in the h2 community: 
> Short of modifying the source code of the Parser class in H2, no, there is not much you can do.
https://github.com/h2database/h2database/issues/3675
